<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Python Chat Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="locale">
    <locale language="Portuguese" country="Brazil"/>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="subject_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>361</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Assunto</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="message_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Mensagem</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Nome </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>551</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Manoel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject_text">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>361</y>
       <width>451</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="message_text">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>391</y>
       <width>451</width>
       <height>61</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="send_button">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>361</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Use Ctrl+Return para enviar"/>
     </property>
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="chat_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>621</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
